controllers/metal3.io/hostfirmwaresettings_controller.go

```diff
@@ -162,6 +163,10 @@ func (r *HostFirmwareSettingsReconciler) Reconcile(ctx context.Context, req ctrl
 	}
 
 	// requeue to run again after delay
+	if meta.IsStatusConditionTrue(info.hfs.Status.Conditions, string(metal3v1alpha1.FirmwareSettingsChangeDetected)) {
+		// If the Spec is different that Status shorten the time between queries to get Ironic updates after cleaning
```

Comment:
This comment is a little bit unclear to me

Reply:
Fixed